tools/internal_ci/linux/grpc_xds_k8s_run_xtest.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## xDS test server/client Docker images
+readonly IMAGE_REPO="gcr.io/grpc-testing/xds-interop"
+
+find_latest() {
+  gcloud container images list-tags --filter=tags\~v1\.\\d\\d\.x "${IMAGE_REPO}/${1}-${2}" '--format=table[no-heading](tags)' | awk -F, '{print $2}' | sort | tail -n 1
```

Comment:
```suggestion
  gcloud container images list-tags --filter='tags~v1\.\d+\.x' --flatten='tags[]' --format='value(tags)' | sort --version-sort | tail -n 1
```

1. `--filter='tags~v1\.\d+\.x'` - a better filter; single quoted to  avoid escapes
2. `--flatten='tags[]' --format='value(tags)'` - just prints the matched tag name. So awk is not needed
3. `sort --version-sort` - see comment to the line 30